package.json

```diff
@@ -20,33 +21,40 @@
   },
   "homepage": "",
   "dependencies": {
-    "@babel/polyfill": "^7.7.0",
+    "@types/pg": "^7.11.2",
+    "@types/supertest": "^2.0.8",
     "body-parser": "^1.19.0",
     "chai": "^4.2.0",
     "cors": "^2.8.4",
+    "custom-env": "^1.0.2",
     "dotenv-config": "^0.1.1",
     "express": "^4.17.1",
     "nodemon": "^1.19.4",
-    "nyc": "^14.1.1"
+    "nyc": "^14.1.1",
+    "pg": "^7.14.0"
```

Comment:
I have a feeling this too should be under `devDependencies`, devDependencies are packages which are used during development, `dependencies` are packages used during runtime.